src/test/java/com/utils/CreatePropertiesFileTest.java

```diff
@@ -0,0 +1,51 @@
+package com.utils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CreatePropertiesFileTest {
+
+    private final String FILENAME = "application.properties";
+    private String propertySource;
+
+    private CreatePropertiesFile classUnderTest;
+    private Properties properties;
+
+    @BeforeEach
+    void setUp() {
+        this.classUnderTest = new CreatePropertiesFile();
+        properties = new Properties();
+        propertySource = "C:/Петко/udemy/java master class/Martin_Project/testResource/test.txt";
+    }
+
+    @Test
+    void whenLoadingProperties_thenExpectAllPropertiesToBeFilled() {
+        classUnderTest.loadPropertiesFromFile(FILENAME);
+    }
+
+    @Test
+    void givenRealProperties_whenLoading_thenNotNullExpectToBeReturned() {
+        assertNotNull(properties);
+    }
+
+    @Test
+    void givenRealName_whenGettingProperty_thenRightPropertyFileToBeExecuted() {
+        assertEquals(classUnderTest.getPropertyName(), FILENAME);
+    }
+
+    @Test
+    void givenPropertyKey_whenGettingProperty_thenRightPropertySourceToBeExecuted() {
```

Comment:
You can retrieve all sorts of properties from the file not only propertySource(by the way what is propertySource ? ). I think you don't fully understand what your method does and what it can be used for